tests/phpunit/api/v4/Action/SqlFunctionTest.php

```diff
@@ -234,6 +234,8 @@ public function testDateFunctions() {
       ->addSelect('QUARTER(birth_date) AS quarter')
       ->addSelect('MONTH(birth_date) AS month')
       ->addSelect('EXTRACT(YEAR_MONTH FROM birth_date) AS year_month')
+      ->addSelect('DAYOFWEEK(birth_date) AS day_number')
+      ->addSelect('DAYNAME(birth_date) AS day_name')
```

Comment:
```suggestion
      ->addSelect('DAYOFWEEK(birth_date):label AS day_name')
```
Should do the same thing right?

Reply:
Hmm - that produces:
```
There was 1 error:

1) api\v4\Action\SqlFunctionTest::testDateFunctions
CRM_Core_Exception: Aliasing field names is not allowed, only expressions can have an alias.
```